components/pricing_modal/content.tsx

```diff
@@ -57,7 +57,7 @@ function Content(props: ContentProps) {
     const products = useSelector(selectCloudProducts);
 
     const isEnterprise = product?.sku === CloudProducts.ENTERPRISE;
-    const isEnterpriseTrial = subscription?.is_free_trial === 'true';
+    const isEnterpriseTrial = isEnterprise && subscription?.is_free_trial === 'true';
```

Comment:
Is this check needed? I believe we can rely on the `isFreeTrial` tag and in the future it will still apply if we decide to launch the option to trial other plans like Professional